### ISSUE TITLE
OpenSlides 2.3 is Always in OfflineMode

### ISSUE THREAD

**Issue by mwaeckerlin** [OpenSlides/openslides-docker#9]
I opened the same ticker in [OpenSlides/OpenSlides #4208](https://github.com/OpenSlides/OpenSlides/issues/4208). There @tsiegleauq commented, I should address to this project. IMHO that's wrong, because I don't use your Dockerfile and I think the problem is absolutely not related to docker. But anyway, I follow his recommendation, here is my problem:

![grafik](https://user-images.githubusercontent.com/4056726/51906824-6023d000-23c5-11e9-803d-05009f0a3532.png)

I try to run OpenSlides 2.3 (the current stable release) in docker.

`Dockerfile`:
```Dockerfile
FROM mwaeckerlin/base
ENV VERSION                     "2.3"
ENV SOURCE                      "https://files.openslides.org/releases/${VERSION}/openslides-${VERSION}.tar.gz"
RUN apk add --no-cache --purge --clean-protected -u python3 python3-dev gcc libc-dev
WORKDIR /tmp
RUN wget -qO- $SOURCE | tar xz
WORKDIR /tmp/openslides-${VERSION}
RUN python3 setup.py install

USER ${RUN_USER}
CMD openslides start --local-installation

EXPOSE 8000
```

Compile it and run:
```bash
docker run -it --rm --name openslides -p 8000:8000 -u root mwaeckerlin/openslides openslides
```

So basically, the installation is as follows:
 - get an Alpine Linux (the same problem occurs in Ubuntu too)
 - install `python3`, `python3-dev`,  `gcc` and `libc-dev`
 - download and extract https://files.openslides.org/releases/2.3/openslides-2.3.tar.gz
 - run `python3 setup.py install`
 - run `openslides start --local-installation`
 - connect to http://localhost:8000
 - enter user: `admin`, password: `admin`

Then you always get the error message: `Offline mode: You can use OpenSlides but changes are not saved.`

What's the problem?

**Comment by mwaeckerlin** [comment]
It's strange: Now it works. I cannot explain it, there is no difference in the setup. Anyways, it's no more reproducible, so I close the ticket.
